src/components/MainRoute.scss

```diff
@@ -8,10 +8,14 @@
 
 .main-content-wrapper {
     flex: 1;
+    position: relative;
+    display: flex;
+    overflow: hidden;
 }
 
 .bottom-navigator-wrapper {
     display: flex;
+    user-select: none;
```

Comment:
Non-cross-platform solution. It's may not work in some browsers. For example, that snippet add maintenance
```
  -webkit-user-select: none; /* Safari */
  -moz-user-select: none; /* Firefox */
  -ms-user-select: none; /* IE10+/Edge */
  user-select: none; /* Standard */
 ```